assets/script/script.js

```diff
@@ -111,3 +116,18 @@ modalCloseBtn.addEventListener('click', () => {
     popupDetails[i].classList.remove('popup-details-active');
   }
 });
+
+function hasUpperCase(str) {
+  return /[A-Z]/.test(str);
+}
+contactSubmitBtn.addEventListener('click', () => {
+  errorMessage.innerHTML = '';
+
+  hasUpperCase(contactFormEmail.value);
+  if (hasUpperCase(contactFormEmail.value)) {
+    errorMessage.innerHTML = 'Please enter lower case characters for the email address';
+  } else {
+    contactForm.submit();
+    contactForm.reset();
+  }
+});
```

Comment:
- [ ] Ouch! this is really close but I noticed the form data is being submitted even though the error message is being displayed and this is because you're not preventing the default action of submitting the form using `  e.preventDefault();.` Check the code below 👇
```suggestion
contactSubmitBtn.addEventListener('click', (e) => {
  e.preventDefault();
  errorMessage.innerHTML = '';

  hasUpperCase(contactFormEmail.value);
  if (hasUpperCase(contactFormEmail.value)) {
    errorMessage.innerHTML = 'Please enter lower case characters for the email address';
  } else {
    contactForm.submit();
    contactForm.reset();
  }
});
```